pkg/kudoctl/cmd/install/install.go

```diff
@@ -248,23 +260,26 @@ func installSingleFrameworkVersionToCluster(name, namespace string, kc *kudo.Cli
 
 // installSingleInstanceToCluster installs a given Instance to the cluster
 // TODO: needs more testing
-func installSingleInstanceToCluster(name, previous string, instance *v1alpha1.Instance, kc *kudo.Client, options *Options) error {
+func installSingleInstanceToCluster(name string, instance *v1alpha1.Instance, kc *kudo.Client, options *Options) error {
 	// Customizing Instance
 	// TODO: traversing, e.g. check function that looksup if key exists in the current FrameworkVersion
 	// That way just Parameters will be applied if they exist in the matching FrameworkVersion
+
+	if _, err := kc.InstallInstanceObjToCluster(instance, options.Namespace); err != nil {
+		return errors.Wrapf(err, "installing instance %s", name)
+	}
+	fmt.Printf("instance.%s/%s created\n", instance.APIVersion, instance.Name)
+	return nil
+}
+
+func applyInstanceOverrides(instance *v1alpha1.Instance, options *Options, isDependencyInstall bool) {
 	// More checking required
 	// E.g. when installing with flag --all-dependencies to prevent overwriting dependency Instance name
-
 	// This checks if flag --instance was set with a name and it is the not a dependency Instance
-	if options.InstanceName != "" && previous == "" {
+	if options.InstanceName != "" && !isDependencyInstall {
```

Comment:
the comment above line 278 is the original message...  it doesn't make sense "it is the not a dependency Instance".  could we reword it.   It seems to be associated with the code that was changed.